en/best-practices.md

```diff
@@ -236,17 +238,18 @@ Eventually, the GTFS Best Practices will become part of the core GTFS Reference.
 
 ### How to check for conformance with these Best Practices?
 
-The Canonical GTFS Schedule Validator checks for compliance against these Best Practices. You can find more about this validation tool on the [validate page](https://gtfs.org/schedule/validate/).
+The Canonical GTFS Schedule Validator checks for compliance against the GTFS Best Practices. Each WARNING corresponds to recommendations that are either explicitly suggested by the GTFS Schedule Reference, using the term “recommend” or “should,” or mentioned in this document.
+You can find more about this validation tool on the [validate page](https://gtfs.org/schedule/validate/).
 
 ### I represent a transit agency. What steps can I take so that our software service providers and vendors follow these Best Practices?
 
 Refer your vendor or software service provider to these Best Practices. We recommend referencing the GTFS Best Practices URL, as well as core Spec Reference in procurement for GTFS-producing software.
 
-### What should I do if I notice a GTFS data feed does not conform to these Best Practices?
+### What should I do if I notice a GTFS data feed does not conform to the GTFS Best Practices?
 
-Identify the contact for the feed, using the [proposed feed\_contact\_email or feed\_contact\_url](https://github.com/google/transit/pull/31/files) fields in *feed_info.txt* if they exist, or looking up contact information on the transit agency or feed producer website. When communicating the issue to the feed producer, link to the specific GTFS Best Practice under discussion. (See ["Linking to this Document"](#linking-to-this-document)).
+Identify the contact for the feed, using the `feed_contact_email` or `feed_contact_url` fields in [feed_info.txt](https://gtfs.org/schedule/reference/#feed_infotxt) if they are provided, or looking up contact information on the transit agency or feed producer website. When communicating the issue to the feed producer, link to the specific GTFS Best Practice that is non-compliant using this document (See ["Linking to this Document"](#linking-to-this-document)) or the Canonical GTFS Schedule Validator [documentation](https://gtfs-validator.mobilitydata.org/rules.html#duplicate_route_name-rule).
```

Comment:
"link to the specific GTFS Best Practice that is non-compliant" is a bit confusing, since you're describing the non-compliant feed, not the best practice. What about "link to the GTFS Best Practice that isn't being followed using this document"

Reply:
Agree. Also should update the `the Canonical GTFS Schedule Validator ` to something like `the appropriate warning in the Canonical GTFS Schedule Validator, which evaluates consistency with the recommended or should statements in the GTFS Spec`... and then might not want to link to a specific rule?

Reply:
We can update to the the list of WARNINGS in the validator instead: https://gtfs-validator.mobilitydata.org/rules.html#WARNING-table